numerics/combinatorics_body.hpp

```diff
@@ -0,0 +1,18 @@
+
+#pragma once
+
+#include "numerics/combinatorics.hpp"
+
+namespace principia {
+namespace numerics {
+
+constexpr int Binomial(int n, int k) {
+  return k == 0 ? 1 : n == k ? 1 : Binomial(n - 1, k - 1) + Binomial(n - 1, k);
```

Comment:
Do we trust the compiler to do memoization on constexpr evaluation?

Reply:
I think there should be a FORCE_INLINE here.

Reply:
Hmm, this triggers C4714.  Needs more work.